contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -945,11 +959,24 @@ pltsql_GetNewTempOidWithIndex(Relation relation, Oid indexId, AttrNumber oidcolu
 	 */
 	Assert(temp_oid_buffer_size > 0);
 
+	/*
+	* debug level logs when trying to generate a new temp object oid from the given catalog.
+	*/
+	elog(DEBUG1, "Generating new oid for tsql temp object from system catalog with oid: %u and column: %d", relation->rd_id, oidcolumn);
```

Comment:
is oidcolumn really interesting enough to log?

Reply:
given the nature of what we are trying to debug (the missing pg_class issue), i would think it makes a little more sense to log some interesting info about the relation. It's guaranteed to be a catalog relation; can we print the relfilenode and anything else that might be relevant?